src/catalog/pgduckdb_schema.cpp

```diff
@@ -7,86 +7,83 @@
+PostgresSchema::CreateTable(duckdb::CatalogTransaction, duckdb::BoundCreateTableInfo &) {
 	throw duckdb::NotImplementedException("CreateTable not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreateView(duckdb::CatalogTransaction transaction, duckdb::CreateViewInfo &info) {
+PostgresSchema::CreateView(duckdb::CatalogTransaction, duckdb::CreateViewInfo &) {
 	throw duckdb::NotImplementedException("CreateView not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreateSequence(duckdb::CatalogTransaction transaction, duckdb::CreateSequenceInfo &info) {
+PostgresSchema::CreateSequence(duckdb::CatalogTransaction, duckdb::CreateSequenceInfo &) {
 	throw duckdb::NotImplementedException("CreateSequence not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreateTableFunction(duckdb::CatalogTransaction transaction, duckdb::CreateTableFunctionInfo &info) {
+PostgresSchema::CreateTableFunction(duckdb::CatalogTransaction, duckdb::CreateTableFunctionInfo &) {
 	throw duckdb::NotImplementedException("CreateTableFunction not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreateCopyFunction(duckdb::CatalogTransaction transaction, duckdb::CreateCopyFunctionInfo &info) {
+PostgresSchema::CreateCopyFunction(duckdb::CatalogTransaction, duckdb::CreateCopyFunctionInfo &) {
 	throw duckdb::NotImplementedException("CreateCopyFunction not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreatePragmaFunction(duckdb::CatalogTransaction transaction, duckdb::CreatePragmaFunctionInfo &info) {
+PostgresSchema::CreatePragmaFunction(duckdb::CatalogTransaction, duckdb::CreatePragmaFunctionInfo &) {
 	throw duckdb::NotImplementedException("CreatePragmaFunction not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreateCollation(duckdb::CatalogTransaction transaction, duckdb::CreateCollationInfo &info) {
+PostgresSchema::CreateCollation(duckdb::CatalogTransaction, duckdb::CreateCollationInfo &) {
 	throw duckdb::NotImplementedException("CreateCollation not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::CreateType(duckdb::CatalogTransaction transaction, duckdb::CreateTypeInfo &info) {
+PostgresSchema::CreateType(duckdb::CatalogTransaction, duckdb::CreateTypeInfo &) {
 	throw duckdb::NotImplementedException("CreateType not supported yet");
 }
 
 duckdb::optional_ptr<duckdb::CatalogEntry>
-PostgresSchema::GetEntry(duckdb::CatalogTransaction transaction, duckdb::CatalogType type,
-                         const duckdb::string &entry_name) {
-	auto &pg_transaction = transaction.transaction->Cast<PostgresTransaction>();
-	return pg_transaction.GetCatalogEntry(type, name, entry_name);
+PostgresSchema::GetEntry(duckdb::CatalogTransaction _catalog_transaction, duckdb::CatalogType _type,
```

Comment:
Why are these underscores necessary? So they don't collide with class members? Seems unnecessary for the
`_catalog_transaction` one.

Reply:
> Why are these underscores necessary? So they don't collide with class members?

Yes exactly, while technically C++ let you shadow class members with function parameters, it can be source of bugs to allow it.

And even though it was not necessary for `_catalog_transaction` I preferred to be consistent for all parameters.